Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -802,9 +808,12 @@ private void RegisterPointers(IMixedRealityInputSource inputSource)
                     return;
                 }
 
+                var inputSystem = CoreServices.InputSystem;
+                IMixedRealityGazeProvider gazeProvider = inputSystem.GazeProvider;
+
                 IMixedRealityPointerMediator mediator = null;
 
-                var mediatorType = CoreServices.InputSystem?.InputSystemProfile.PointerProfile.PointerMediator.Type;
+                var mediatorType = inputSystem?.InputSystemProfile.PointerProfile.PointerMediator.Type;
```

Comment:
Another spot where we could use the cached `inputSystemProfile`

Reply:
Done!